docs/repo-docs/reference/configuration.mdx

```diff
@@ -117,6 +117,47 @@ To help with incremental migration or in situations where you can't use the `pac
   environment variable.
 </Callout>
 
+### `cacheDir`
+
+Default: `".turbo/cache"`
+
+Specify the filesystem cache directory.
+
+```jsonc title="./turbo.json"
+{
+  "cacheDir": ".turbo/cache"
+}
+```
+
+### `daemon`
+
+Default: `false`
+
+Turbo can run a background process to pre-calculate values used for determining work that needs to be done. This standalone process (daemon) is an optimization, and not required for proper functioning of turbo.
```

Comment:
```suggestion
Turborepo runs a background process to pre-calculate values used for determining work. This standalone process (daemon) is a performance optimization, and not required for proper functioning of `turbo`.
```

Reply:
this was lifted directly (unchanged) from the existing docs (which you wrote? haha) - should I update there too?

Reply:
Yes, definitely. 😄